src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      action="/action_page.php"
+      method="post"
+      class="form"
+    >
+      <div class="form__big-icon">
+        <img
+          class="form__big-img"
+          src="../src/images/Search.svg"
+          alt="icon search"
+        >
+      </div>
+      <input
```

Comment:
add name for each input, as I mentioned [here](https://github.com/mate-academy/layout_search-bar-airbnb/pull/3567#discussion_r1306619492)